pkg/commands/workload.go

```diff
@@ -200,6 +205,34 @@ func (opts *WorkloadOptions) Validate(ctx context.Context) validation.FieldError
 		errs = errs.Also(validation.Enum(opts.Output, flags.OutputFlagName, []string{printer.OutputFormatJson, printer.OutputFormatYaml, printer.OutputFormatYml}))
 	}
 
+	// validating sources as the source options are mutually exclusive
+	if opts.MavenArtifact != "" || opts.MavenVersion != "" || opts.MavenGroup != "" || opts.MavenType != "" {
+		mavenSource = true
+	}
+	for _, p := range opts.ParamsYaml {
+		kv := parsers.DeletableKeyValue(p)
+		if len(kv) != 1 {
+			if kv[0] == cartov1alpha1.WorkloadMavenParam {
+				mavenSource = true
+				break
+			}
+		}
+	}
```

Comment:
You would want to check this if `mavenSource` was not set to true in above `if` conditional?

Reply:
if either of the mavan sources are there in the command, we need to show that message and hence we would be ok even if the value was set as true before.

Reply:
if `mavenSource` is already true, why would you loop again to make it `true`?